truffle/contracts/Owned.sol

```diff
@@ -0,0 +1,25 @@
+pragma solidity 0.5.10;
+
+contract Owned {
+    
+    address public owner;
+    
+    event LogOwnerChanged(address newOwner);
+    
+    modifier _onlyOwner {
+        require(msg.sender == owner);
+        _;
+    }
+    
+    constructor() public {
+        owner = msg.sender;
+    }
+    
+    function changeOwner(address newOwner) internal _onlyOwner returns(bool) {
+        owner = newOwner;
+        
```

Comment:
It might be smart to check that the input is a valid address before throwing control away.